beacon-chain/db/filesystem/pruner.go

```diff
@@ -169,6 +182,15 @@ func (p *blobPruner) tryPruneDir(dir string, pruneBefore primitives.Slot) (int,
 	return len(scFiles), nil
 }
 
+func idxFromPath(fname string) (uint64, error) {
+	fname = path.Base(fname)
+	parts := strings.Split(fname, ".")
+	if len(parts) != 2 {
+		return 0, errors.New("not a blob ssz file")
```

Comment:
Someone could in theory have some other file with 2 parts that isn't an ssz. Wouldn't it be safer to check if it has the extension?

Reply:
This is called downstream of the ssz filter (`scFiles := filter(entries, filterSsz)`), but we could double-check it here.